regex_macros/src/lib.rs

```diff
@@ -115,9 +113,6 @@ impl<'a> NfaGen<'a> {
         );
         let prefix_anchor = self.prog.anchored_begin;
 
-        // let prefix_lit = Rc::new(self.prog.prefix.as_bytes().to_vec());
-        // let prefix_bytes = self.cx.expr_lit(self.sp, ast::LitBinary(prefix_lit));
-
         // let check_prefix = self.check_prefix();
```

Comment:
Not particularly. I'm not a huge fan of keeping around commented out code, especially without an explanation.